src/pipeline/activator/cog-clip/README.md

```diff
@@ -0,0 +1,297 @@
+# Cog Clip module
+
+Given a STAC Item Id and the STAC Collection, clips Features from it an puts into the Target Collection.
+
+## Demo
+
+A picture of the original scene and two clipped out polygons.
+
+<img width="502" alt="image" src="https://user-images.githubusercontent.com/4929546/104251306-6f2eac00-543d-11eb-9ad3-794aae05b7bc.png">
+
+```bash
+[info] running com.azavea.nasa.hsi.Main clip --help
+[ioapp-compute-0] ERROR c.a.n.hsi.Main - Usage:  clip --source-collection-id <string> --source-item-id <string> --source-asset-id <string> --target-collection-id <string> [--target-layer-id <string>] [--features <{ "type": "FeatureCollection", "features": [<features>] }>] [--stac-api-uri <uri>] [--target-s3-uri <Amazon S3 URI>] [--threads <integer>] [--with-gdal] [--force]
+
+Clip extents from COG as provided by GeoJSON Feature Collection
+
+Options and flags:
+    --help
+        Display this help text.
+    --source-collection-id <string>
+        Source collectionId.
+    --source-item-id <string>
+        Source itemId.
+    --source-asset-id <string>
+        Source assetIt.
+    --target-collection-id <string>
+        Target collectionId.
+    --target-layer-id <string>
+        Target layerId.
+    --features <{ "type": "FeatureCollection", "features": [<features>] }>
+        Feature Collection of features to clip from COG
+    --stac-api-uri <uri>
+
+    --target-s3-uri <Amazon S3 URI>
+
+    --threads <integer>
+        Number of threads
+    --with-gdal
+        Uses GDAL for raster reads.
+    --force
+        Force reingest StacItem even though this it is already present in the catalog.
+
+Environment Variables:
+    FEATURES=<{ "type": "FeatureCollection", "features": [<features>] }>
+        Feature Collection of features to clip from COG.
+    STAC_API_URI=<uri>
+    
+    TARGET_S3_URI=<Amazon S3 URI>
+    THREADS=<integer>
+        Number of threads.
+```
+
+## Testing Instructions
+
+* Run a local Franklin instance.
+* <details> 
+     <summary>Create `aviris-l2-cogs` collection.</summary>
```

Comment:
It's not clear how to create this collection. I'd consider including a specific `curl` command here.

Reply:
@rbreslow I added them, but it is still not very convenient since input json files are pretty large.